certified-connectors/Witivio/apiDefinition.swagger.json

```diff
@@ -3,7 +3,12 @@
   "info": {
     "title": "Witivio",
     "description": "Witivio 365 and Microsoft Flow.",
-    "version": "1.0"
+    "version": "2.0",
+    "contact": {
+      "name": "Witivio",
+      "url": "https://witivio.com",
```

Comment:
@tdechanterac Please add domain to your website, like below:

`"url": "https://www.witivio.com"`

Reply:
I've modified as well all the links with the proper links.